operators/kepler-operator/0.4.0/metadata/annotations.yaml

```diff
@@ -0,0 +1,17 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: kepler-operator
+  operators.operatorframework.io.bundle.channels.v1: alpha
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.23.0
+  operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
+  operators.operatorframework.io.metrics.project_layout: go.kubebuilder.io/v3
+
+  # Annotations for testing.
+  operators.operatorframework.io.test.mediatype.v1: scorecard+v1
+  operators.operatorframework.io.test.config.v1: tests/scorecard/
+
+  #Annotattions for OpenShift version
+  com.redhat.openshift.versions: "v4.9-v4.10"
```

Comment:
I see the CI fire tests for OCP v 4.8 - 4.12. Since I've specified in the annotation `  com.redhat.openshift.versions: "v4.9-v4.10"` should it not be tested only on 4.9 and 4.10 ?